app/models/account/credit.rb

```diff
@@ -1,3 +1,9 @@
 class Account::Credit < ApplicationRecord
   include Accountable
+  SUBTYPES = [
```

Comment:
Could `SUBTYPES` be an array, and the human-friendly version defined as translations?

```ruby
  SUBTYPES = %w[ mastercard visa american_express discover ]
```